src/lib/numbers.js

```diff
@@ -0,0 +1,63 @@
+function add(a, b) {
+  return a + b;
+}
+
+function subtract(a, b) {
+  return a - b;
+}
+
+function multiply(a, b) {
+  return a * b;
+}
+
+function divide(a, b) {
+  if (a == 0) {
+    return 0;
+  }
+  return a / b;
+}
+
+function power(a, b) {
+  // eslint-disable-next-line no-restricted-properties
+  return Math.pow(a, b);
```

Comment:
No ned for the eslint exception here, when hovering over the error it even gives you the answer!
```
  return a ** b;
```